flutter/pubspec.yaml

```diff
@@ -13,20 +13,21 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  sentry: ">=4.0.0 <5.0.0"
-  package_info: ^0.4.0
+  # this should point to pub
+  sentry:
+    path: ../dart
+  package_info: ^2.0.0
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  mockito: ^4.1.3
-  test: ^1.15.7
-  yaml: ^2.2.1 # needed for version match (code and pubspec)
-  pedantic: ^1.9.2
+  mockito: ^5.0.0-nullsafety.7
+  yaml: ^3.0.0 # needed for version match (code and pubspec)
+  pedantic: ^1.10.0
 
-dependency_overrides:
-  sentry:
-    path: ../dart
+# dependency_overrides:
+#  sentry:
+#    path: ../dart
```

Comment:
this helps developing and to debug, I'd keep as it is till we get a full null safety migration